vars/test.yml

```diff
@@ -415,6 +415,11 @@ flatcar_python3_download_url: "https://downloads.activestate.com/ActivePython/re
 #
 # http_proxy_vm: "http://myproxy.company.com:8080"
 
+# For Linux Mainline Kernel Upgrade only in Ubuntu:
+# e.g., 'latest', '6.10', '6.11-rc1'.
+#
+# upgrade_kernel_version: latest
```

Comment:
If mainline kernel versions all have a 'v' prefix, please suggest user to use v6.10, v6.11-rc1, etc